sample.env

```diff
@@ -22,4 +22,4 @@ REACT_APP_GEM_COINIFY_SUPPORTED_COINS=https://api.gem.co/institutions/coinify/su
 REACT_APP_GEM_WYRE_SUPPORTED_COINS=https://api.gem.co/institutions/wyre/supported_currencies
 REACT_APP_GEM_ASSET_LOGO=https://gem-widgets-assets.s3-us-west-2.amazonaws.com/currencies/crypto/
 REACT_APP_GEM_ENV=production
-REACT_APP_GEM_API_KEY=17baca45f492df6b8974d4f832a659c92644acafea3284cfb0d52550dd3bde1a
+REACT_APP_GEM_API_KEY=bb4164a72246dae1e03010d664d6cdae4e19b2554de02e3bf6c3cd30aa7e359e
```

Comment:
this should not be changing in this PR

Reply:
Pulled this off the newest commit, reverting back to previous